src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs

```diff
@@ -2476,27 +2482,37 @@ private protected string GetInternalToolTipText(int rowIndex)
             return toolTipText;
         }
 
-        private string GetToolTipText(int rowIndex)
+        /// <summary>
+        /// This method gets the tooltip text for cells.
```

Comment:
These API passing just a row but no column index are very strange.
```suggestion
        /// Gets the tooltip text for current cell.
```